### ISSUE TITLE
Todos

### ISSUE THREAD

**Issue by aidangomez** [1vn/mnist-tf-tutorial#1]
Good job man! I'm impressed with how far you got. Good code design.

Todo:
- [ ] remove data reading from your trainer
- [ ] add a text file with the performance comparisons between different depths, dropout vs. no dropout, etc.
- [ ] should use a loop and a 'depth' hyperparam instead of manually defining the depth
- [x] consider using tf.nn.dense instead of the manual stuff (but I'm glad you did this since you'll actually know what a dense layer is doing.)
- [ ] some comments to clean up
- [ ] LSTM isn't done, talk to Navie about his implementation and team up to get something that trains
